src/app/(app)/_components/header/index.tsx

```diff
@@ -4,14 +4,14 @@ export default function Page() {
   return (
     <div className="relative">
       <div className="flex items-end justify-center">
-        <div className="offside relative z-10 flex text-[8rem]">
+        <div className="offside relative z-10 flex text-[4rem] md:text-[8rem]">
           <h1 className="neon_blue">BeCreate</h1>
           <h1 className="neon_orange rotate-12">!</h1>
         </div>
```

Comment:
h1タグは1ページに1つのほうがSEO的に良いので以下のように1つにしてください！
[MDN 1 つのページに複数の h1 要素を使用しない](https://developer.mozilla.org/ja/docs/Web/HTML/Element/Heading_Elements#1_%E3%81%A4%E3%81%AE%E3%83%9A%E3%83%BC%E3%82%B8%E3%81%AB%E8%A4%87%E6%95%B0%E3%81%AE_h1_%E8%A6%81%E7%B4%A0%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%97%E3%81%AA%E3%81%84)


```suggestion
        <h1 className="offside relative z-10 flex text-[4rem] md:text-[8rem]">
          <span className="neon_blue">BeCreate</span>
          <span className="neon_orange rotate-12">!</span>
        </h1>
```

Reply:
勉強になります！！